problems/getCountriesSortedByPopulation.js

```diff
@@ -8,7 +8,10 @@
 */
 
 function getCountriesSortedByPopulation() {
-
-}
-
+    let arr = [];
+    return arr.sort(function(country, population) {
```

Comment:
Close, sort takes a comparison callback. So intead of country and population, they are both country objects. a.population - b.population